GATEWAY/src/pages/virtual-tiles/virtual-tiles.ts

```diff
@@ -10,15 +10,18 @@ import { Application, Device, UtilsService, VirtualTile } from '../../providers/
   See http://ionicframework.com/docs/v2/components/#navigation for more info on
   Ionic pages and navigation.
 */
+
 @Component({
   selector: 'page-virtual-tiles',
   templateUrl: 'virtual-tiles.html'
 })
 export class VirtualTilesPage {
   devices: Device[];
-	applicationTitle: string;
+  applicationTitle: string;
   virtualTiles: VirtualTile[];
   activeApp: Application;
+  pairedVirtualTiles = [];
```

Comment:
Wouldn't it work to map over the tiles to check if paired or not instead? Seems redundant to store them twice.